lib/OpenQA/Schema/Result/ScheduledProducts.pm

```diff
@@ -590,12 +592,13 @@ sub _generate_jobs {
     }
 
     $ret = _sort_dep($ret);
+
```

Comment:
Still one unrelated blank line change but if this is the only change then it's OK to keep as is :)